docs/release-notes/next/PT-165979853-extract-log-info.md

```diff
@@ -0,0 +1 @@
+* Add a script for extracting session-specific log info from aeternity.log files
```

Comment:
Is scripts/log_xtract.escript shipped to the users as part of the package? Is it user oriented? It does not look to me so I see no sense in having a release note line for this.

Reply:
Although it could be used by users, I think this is mainly for developer's convenience so probably the release notes line will be misleading

Reply:
Yes, it's mainly for developers, although, as Dimitar says, it could perhaps become useful for advanced users or app developers.